# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1644060095 -3600
# Parent  f9fec58dc6568646f51d72a930c55fbcc173178b
Misc: Add more logging to update check.

diff --git a/mail/base/content/aboutDialog.js b/mail/base/content/aboutDialog.js
--- a/mail/base/content/aboutDialog.js
+++ b/mail/base/content/aboutDialog.js
@@ -121,17 +121,21 @@ async function BetterbirdUpdateCheck(win
       let appLocale = Services.locale.appLocalesAsBCP47[0];
       for (let l of lines) {
         let parts = l.split(",");
         // Format is locale,version,buildid.
         if (version.startsWith(parts[1]) && appLocale == parts[0]) {
           if (parts[2] > buildID) {
             needsUpdate = true;
             console.info(
-              `Betterbird update available for locale ${appLocale}, new build ${parts[2]}`
+              `Betterbird update available for locale ${appLocale}, new build ${parts[2]}, current build ${buildID}`
+            );
+          } else {
+            console.info(
+              `Betterbird update-to-date for locale ${appLocale}, published build ${parts[2]}, current build ${buildID}`
             );
           }
           break;
         }
       }
     } catch (ex) {}
   }
 
diff --git a/mail/components/MailGlue.jsm b/mail/components/MailGlue.jsm
--- a/mail/components/MailGlue.jsm
+++ b/mail/components/MailGlue.jsm
@@ -705,17 +705,21 @@ MailGlue.prototype = {
       let appLocale = Services.locale.appLocalesAsBCP47[0];
       for (let l of lines) {
         let parts = l.split(",");
         // Format is locale,version,buildid.
         if (version.startsWith(parts[1]) && appLocale == parts[0]) {
           if (parts[2] > buildID) {
             needsUpdate = true;
             console.info(
-              `Betterbird update available for locale ${appLocale}, new build ${parts[2]}`
+              `Betterbird update available for locale ${appLocale}, new build ${parts[2]}, current build ${buildID}`
+            );
+          } else {
+            console.info(
+              `Betterbird update-to-date for locale ${appLocale}, published build ${parts[2]}, current build ${buildID}`
             );
           }
           break;
         }
       }
     } catch (ex) {}
     if (!needsUpdate) {
       return;
